app/models/concerns/relatable.rb

```diff
@@ -18,8 +18,8 @@ def self.counter_names
       end
     end
 
-    def self.privacy_adjusted_count_columns
-      count_columns.map do |column|
+    def self.privacy_adjusted_count_columns(except: [])
+      (count_columns - except).map do |column|
```

Comment:
This concern is rather widely used throughout the model. Can you provide some specs to test the new behavior?

Reply:
I added some minimal tests. This is also tested indirectly by existing tests for `.table_data`.